src/components/Plugins/PluginCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { ExternalLink } from "lucide-react";
+import { Plugin } from "../../interfaces/Plugin";
+
+type PluginCardProps = {
+  plugin: Plugin;
+}
+export function PluginCard({plugin}:PluginCardProps) {
+	return <section className="border gap-4 flex flex-col max-w-[314px]  hover:scale-[1.01]   p-4 pt-6 m-auto w-full border-[#27272A] rounded-md h-[300px]">
+		<div className="flex-1 flex flex-col gap-4">
+			<div>
+				<div>
+					<h1 className="text-white text-[22px] font-semibold">{plugin.name}</h1>
+					{/* 
+      // TODO: should add and fetch data about user using the userId or a github handle linked to it
+    */}
+				</div>
+			</div>
+			<div className="text-white text-sm">
+				{plugin.description}
+			</div>
+			<div className="w-full">
+				<img className="w-full"  src={plugin.imageURL} />
```

Comment:
are we sure the plugin's image wouldn't be enormously big ? what abt adding a fixed max-width ?